src/rosdiscover/recover/symbolic.py

```diff
@@ -122,21 +124,26 @@ def __str__(self) -> str:
         return self.value
 
     @classmethod
-    def from_name(cls, name: str) -> SymbolicValueType:
-        name_to_type = {
+    def name_to_type(cls):
```

Comment:
what's going on here? why no type annotation?

if you want this to be a field on the class, it makes sense to mark it as a @property

Reply:
fixed